tests/cmd/login/mod.rs

```diff
@@ -0,0 +1,22 @@
+use crate::VirtualFileSystem;
+
+#[test]
+fn fixture() {
+    let vfs = VirtualFileSystem::empty().with_virtual_home();
+
+    crate::cli!()
+        .arg("login")
+        .arg("--url")
```

Comment:
not sure if possible, but this would be more legible:
```
crate::cli!()
        .arg("login")
        .args("--url https://org.jfrog.io/artifactory")
        .args("--username foo")
```

Reply:
not possible :/ one would have to pass a slice to `args` which is not really better imo